.travis.yml

```diff
@@ -15,4 +15,6 @@ matrix:
 script: bundle exec rake $TASK
 branches:
   only: master
-before_install: ruby -e "File.write('Gemfile.lock', File.read('Gemfile.lock').split('BUNDLED WITH').first)"
+before_install:
+  - gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
+  - gem install bundler -v '< 2'
```

Comment:
faster / simpler:
`before_install: ruby -e "File.write('Gemfile.lock', File.read('Gemfile.lock').split('BUNDLED WITH').first)"`

Reply:
actually in #314 I've removed it - all builds works fine